db/migrate/20171207140915_create_announcements.rb

```diff
@@ -1,3 +1,4 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Since [Ruby 2.0](https://www.ruby-lang.org/en/news/2013/02/24/ruby-2-0-0-p0-is-released/) these lines aren't needed:

> The UTF-8 default encoding, which make many magic comments omissible

We should consider enabling the `Style/Encoding` cop which now warns when these magic comments are present.